best-practices.md

```diff
@@ -113,6 +114,27 @@ But the stronger recommendation is to host a STAC Browser on your own domain, an
 design to look and feel like your main web presence. STAC aims to be decentralized, so each STAC-compliant data catalog 
 should have its own location and just be part of the wider web.
 
+### Requester Pays
+
+It is very common that large, freely available datasets are set up with a 'requester pays' configuration. This is an option 
+[on AWS](https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html)) and [on 
+Google Cloud](https://cloud.google.com/storage/docs/requester-pays), that enables data providers to make their data freely
+available to everyone, while placing the cost of '[egress](https://www.hostdime.com/blog/data-egress-fees-cloud/)' 
+on to the user. For popular datasets that are large in size the egress costs can be substantial, to the point where much
+less data would be available if the cost of distribution was always on the data provider.
+
+For data providers using STAC with requester pays buckets there are two main recommendations:
+
+1) Put the STAC JSON in a separate bucket that is public for everyone, not requestor pays. This enables the STAC metadata
+to be far more crawlable and searchable, but the cost of the egress of STAC files should be miniscule compared to that of
+the actual data. The STAC community can help you work with cloud providers for potential free hosting if you are doing open
+data as requestor pays and aren't able to pay the costs of a completely open STAC bucket, as they are most all supportive of
+STAC.
+2) Don't use `http://` style urls if the provider has a specific protocol (specifically `s3://` on AWS and `gs://` on Google
```

Comment:
```suggestion
2) For Asset href values to resources in a requestor pays bucket, use the cloud provider-specific protocol (e.g., `s3://` on AWS and `gs://` on Google Cloud) instead of an `https://` url. 
```